todo/src/components/Todo.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { toggleComplete } from './../actions';
+
+class Todo extends React.Component {
+
+    constructor(props) {
+        super(props);
+    }
+    
+    checkComplete = e => {
+        e.preventDefault();
+        this.props.toggleComplete(this.props.todo);
+    }
+
+    render() {
+        console.log(this.props);
+        return (
+            <li  onClick={() => this.props.checkComplete(this.props.todo)}>
+                
+                {this.props.name} - Completed: {this.props.completed.toString()}<span className="fa fa-close"></span>
+            </li>
+        );
+    }
+    
+};
+
+const mtsp = () => {
+    return {}
+}
+
+export default connect(mtsp, { toggleComplete: toggleComplete })(Todo);
```

Comment:
You don't have to have Redux connected to every component if you don't need it on every component! It's OK, but you could also just pass the action creator from a parent to a child component, call it in the child and pass through what you need. 

Let me know if you have any questions about this, and try playing around with it if you have a chance!